examples/js/archive.js

```diff
@@ -57,9 +66,19 @@ function extractKey() {
   }
 }
 
-let imageCount = 0;
-let fetchedFrom;
+// next btn
+function handleNext() {
```

Comment:
Hi, @7malikk -- for these `handleNext` and `handlePrev`, was there a reason you didn't define them in pagination.js? Wouldn't they be better in there too?

Reply:
Ah, i see. you get `currPage` as the output, which is a number? But couldn't we track the current page inside of the Pagination class, and just tell it to go to the next or prev page, and then as for the current page's images? All the knowledge of which page we're on could be managed inside the class, you know?

Reply:
Will do @jywarren Thank you!